source/Calamari/Integration/Processes/Semaphores/SystemSemaphoreManager.cs

```diff
@@ -22,9 +22,15 @@ public SystemSemaphoreManager(ILog log, TimeSpan initialWaitBeforeShowingLogMess
             this.initialWaitBeforeShowingLogMessage = (int)initialWaitBeforeShowingLogMessage.TotalMilliseconds;
         }
 
-#if NET40
         public IDisposable Acquire(string name, string waitMessage)
         {
+            return CalamariEnvironment.IsRunningOnWindows
+                ? AcquireSemaphore(name, waitMessage)
+                : AcquireMutex(name, waitMessage);
```

Comment:
Why can we not just use the Mutex on both Windows and Linux?  It's global right?